product/urls.py

```diff
@@ -1,5 +1,6 @@
-from django.urls import path
+from django.urls   import path
+from .views        import ProductListView
 
 urlpatterns = [
-
+    path('/product-list', ProductListView.as_view()),
```

Comment:
다음과 같이 urlpattern을 설정하면 product list를 보내주는 API의 urlpattern이 product/product-list가 됩니다.
```suggestion
    path('', ProductListView.as_view()),
```
위와 같이 수정하면 /product 만으로도 제품들의 정보를 받아 올 수 있고 더 RESTfull해 집니다.

Reply:
감사합니당 경훈님💡